doc.js

```diff
@@ -139,6 +139,17 @@ define('doc', ['event'], function(event) {
 				return "";
 			},
 
+			'prepend' : function(value) {
+				this.each(function(el) {
+					if(typeof value ==='string') {
+						el.insertAdjacentHTML('afterbegin', value);
+					} else {
+						el.insertAdjacentElement('afterbegin', value);
```

Comment:
What if I pass a doc element here? I think you need to iterate through the doc element in order to prepend everything that has been selected through the querySelector.

@mottam